### PR TITLE
make DataTable doc

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: pytools/datatable.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
 from pytools import Record
 
 
+__doc__ = """
+.. autoclass:: DataTable
```
Comment (inducer) [inline pytools/datatable.py:5]: This probably needs an `.. automodule:: pytools.datatable` to do something.

Comment (matthiasdiener) [reply pytools/datatable.py:8]: Thanks, added!

File: pytools/datatable.py [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
 
     def __init__(self, column_names, column_data=None):
         """Construct a new table, with the given C{column_names}.
 
-        @arg column_names: An indexable of column name strings.
-        @arg column_data: None or a list of tuples of the same length as
-          C{column_names} indicating an initial set of data.
+        :arg column_names: An indexable of column name strings.
+        :arg column_data: None or a list of tuples of the same length as
+         C{column_names} indicating an initial set of data.
```
Comment (inducer) [inline pytools/datatable.py:30]:
```suggestion
             *column_names* indicating an initial set of data.
```

**Comment by inducer** [comment]
Thx!

**Pull request merged**